<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryConfigPage</class>
 <widget class="QWidget" name="HistoryConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabView">
      <attribute name="title">
       <string>Commit list</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Details</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <widget class="QWidget" name="page">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="StringSelectorWidget" name="widget" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Layout</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDetails">
      <attribute name="title">
       <string>Commit details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkViewCommitDetails">
            <property name="text">
             <string>View detailed commit message</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Overview list</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="StringSelectorWidget" name="sswDetailsOverview" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDiffView">
      <attribute name="title">
       <string>Diff Viewer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Position</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <widget class="QRadioButton" name="chkDiffAndHistoryBottom">
            <property name="text">
             <string>View differences below history list right of details</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="chkDiffRight">
            <property name="text">
             <string>View differences right of history list and details</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="chkDiffAboveDetails">
            <property name="text">
             <string>View differences right of history, but above details</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Behaviour</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkDontExpandHunks">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Don't expand hunks</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkPreferRawDiffViewer">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Prefer raw difference view</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StringSelectorWidget</class>
   <extends>QWidget</extends>
   <header>libMacGitverCore/Widgets/StringSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
